src/ts/features/block-quotes.ts

```diff
@@ -0,0 +1,51 @@
+import {Feature, Settings} from '../utils/settings'
+
+export const config: Feature = {
+    id: 'blockquotes',
+    name: 'Blockquote Support',
+    settings: [{type: 'string', id: 'blockquote_prefix', initValue: '> ', label: 'Blockquote Prefix'}],
+    description:
+        'If you start a block with these characters, custom styles will be added to show the block as a blockquote.',
+}
+
+async function blockquotes() {
+    const blocks = document.querySelectorAll('div.roam-block')
```

Comment:
I like how https://github.com/roam-unofficial/roam-toolkit/pull/63/files#diff-e8280a2ce440ae79385e21c80dab2401 defines dedicated list of selectors, I think it'd be a good idea to steal that for selectors you use :)